ui/src/app/utils/access-utils.tsx

```diff
@@ -60,8 +60,8 @@ export const redirectToNiH = (): void => {
   AnalyticsTracker.Registration.ERACommons();
   const url = serverConfigStore.get().config.shibbolethUiBaseUrl + '/login?return-url=' +
       encodeURIComponent(
-        window.location.origin.toString() + NIH_CALLBACK_PATH + '?token=<token>');
-  window.open(url, '_blank');
+        window.location.origin.toString() + NIH_CALLBACK_PATH + '?token={token}');
```

Comment:
Another fix that may work or might just be necessary anyways is encoding the redirect URI again. 

i.e. 

```
window.open(serverConfigStore.get().config.rasLogoutUrl + encodeURIComponent(url), '_blank');
```

Reply:
do you mean keep using` <>`, but encode "%3C and "%3E again?

Reply:
If {} works, that's fine. Yes, with <> escape twice is what I meant